src/stdlib/base.bt

```diff
@@ -1166,6 +1165,18 @@ macro signal_thread(expr) {
 //   printf("PS1: %s\n", str(*uaddr("ps1_prompt")));
 // }
 // ```
+macro uaddr(sym) {
+  let $addr = (uint64)__builtin_uaddr(sym);
+
+  // There are three types of ELF that use uprobe: PIE, EXEC, and dynamic
+  // libraries. Among them, PIE and dynamic libraries need to correct the
+  // symbol address according to the actual load address.
+  if (!__builtin_elf_is_exe) {
```

Comment:
```suggestion
  if comptime (!__builtin_elf_is_exe) {
```
Let me know if this doesn't work but this should strip out the call  to `__bpf_task_map_file_min_addr` in cases where this isn't true